packages/mdc-menu/foundation.js

```diff
@@ -137,6 +142,12 @@ class MDCMenuFoundation extends MDCFoundation {
     this.anchorMargin_ = {top: 0, right: 0, bottom: 0, left: 0};
     /** @private {?AutoLayoutMeasurements} */
     this.measures_ = null;
+    /** @private {number} */
+    this.selectedIndex_ = -1;
+    /** @private {boolean} */
+    this.rememberSelection_ = false;
+    /** @private {boolean} */
+    this.keyDownWithinMenu_ = false;
```

Comment:
this keeps track of whether the user *currently* has a keydown interaction happening on the `mdc-menu` component?

These sorts of fields can be vague...so add a comment about exactly what it is this field represents